.nvmrc

```diff
@@ -0,0 +1 @@
+4.3.0
```

Comment:
I know it's tiny, but as it's not related to the A/B test, best off in a separate PR perhaps?